lib/phlex/view.rb

```diff
@@ -182,11 +182,7 @@ def helpers
 			@_view_context
 		end
 
-		def fetch_attrs(**kwargs)
-			Phlex::ATTRIBUTE_CACHE[kwargs.hash] || _attributes(kwargs)
-		end
-
-		def _attributes(attributes, buffer: +"")
+		def process_attributes(buffer: +"", **attributes)
```

Comment:
Let's change the `buffer` to a positional here so it can't be overridden by a `buffer` attribute. I think this will mean we need to update calls from `Phlex::HTML` to put `**` before the attributes they pass in.

```suggestion
		def process_attributes(buffer = +"", **attributes)
```